src/NuGet.Core/NuGet.Packaging/PackageCreation/Authoring/ManifestReader.cs

```diff
@@ -317,7 +317,7 @@ private static List<ManifestFile> ReadFilesList(XElement xElement)
                 files.AddRange(srcElement.Value.Trim(';').Split(';').Select(s => 
                     new ManifestFile
                     {
-                        Source = s.SafeTrim().TrimStart(slashes),
```

Comment:
Cool, as long as NuGet/Home#2749 does not regress (ideally there are tests for it), then we're fine.